src/Web/View/Render.hs

```diff
@@ -79,12 +79,12 @@ renderText' c vw =
   addCss :: [Text] -> Text -> Text
   addCss [] cnt = cnt
   addCss css cnt = do
-    renderLines (renderContent 2 $ styleElement css) <> "\n\n" <> cnt
+    renderLines (renderContent 0 $ styleElement css) <> "\n\n" <> cnt
```

Comment:
This is going to need merging with the new main, which has more explicit newlines

Reply:
Can do, your fix makes the intention clear.